scripts/imgtool/util.py

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env python3
+#
+# Copyright 2020 JUUL Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def based_str_to_int(value):
```

Comment:
No need to write all of this. Just use `int(string, 0)`.

Reply:
The behavior for octals would be different, `int("077", 0)` will results in: "ValueError: invalid literal for int() with base 0: '077'", so one would need to write `int("0o77", 0)`, but since it's not written anywhere how one should supply numbers I will change it...

Reply:
Done